### PR TITLE
Update heroku info in readme

### PR DESCRIPTION
**Pull request by benmathes**
* some more explanation on some steps
* using `heroku local` instead of `foreman`. `foreman` required `ruby`, `rvm`, and `gem`s, which is no longer needed. `heroku local -f Procfile.local` is interchangeable with `foreman start Procfile.local`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,54 +126,58 @@ With Postgres.app: https://docs.djangoproject.com/en/2.0/ref/contrib/gis/install
...
-    $ rvm reinstall $current_ruby_version --with-openssl-dir=`brew --prefix openssl`
+## Pull Requests and Heroku Builds.
 
-    # To run your dev server use
-    $ foreman start -f $git_root/Procfile.local
+
+Every time you create a PR, heroku will kick off a build to see if it can create
+a server running the version of your PR. You can see the builds here:
+
+https://dashboard.heroku.com/pipelines/514a9ea9-75d3-4056-81cc-24b8aebd5592
```
Comment (ericsandine) [inline README.md]: This reminds me, we need to think about the public vs private repo. Right now this repo is public so anyone could kick off PRs and eat through money on heroku.

Comment (ericsandine) [reply README.md]: I think we get unlimited private repos as a non-profit, might have to confirm that, but we may want to move it private and for the public repo later.

**Comment by ericsandine** [comment]
Oh right! I forgot the heroku CLI just bundled that all together.

Sidenote: looks like they're using a Go implementation of foreman https://github.com/ddollar/forego

File: README.md (hunk 1 above)
Comment (benmathes) [reply README.md]: I think we're ok. LMK if I'm missing anything in my thinking, @ericsandine 
* Anyone who can see that URL has to be able to log in.
* We'd have to approve a PR to kick off a build.

**Pull request merged**